source/Octo.Tests/Octo.Tests.csproj

```diff
@@ -12,14 +12,21 @@
     <ProjectReference Include="..\Octo\Octo.csproj" />
   </ItemGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <NoWarn>CS0618</NoWarn>
```

Comment:
Almost all the repository methods have the `Obsolete` warning for not using the overload with the `CancellationToken`